en/08_Changelogs/6.0.0.md

```diff
@@ -209,12 +211,54 @@ The one change we specifically want to call out is for [`ModelData::obj()`](api:
 
 See the [full list of removed and changed API](#api-removed-and-changed) to see all of the API with updated typing.
 
-### Other new features and small changes {#other-new-features}
```

Comment:
Unrelated: changed "Other new features and small changes" to "Other new features" to match the link in the table of contents, and because "other changes" is its own section, so this section shouldn't have "small changes" that aren't "other new features".